src/cirrus/lib2/statedb.py

```diff
@@ -49,10 +93,12 @@ def __init__(
         self.table_name = table_name
         self.table = self.db.Table(table_name)
 
-        if eventdb:
-            self.eventdb = eventdb
-        else:
-            self.eventdb = EventDB(session=session)
+    @classmethod
+    def get_singleton(cls):
+        """DESCRIBE WHY"""
```

Comment:
```suggestion
        """When running in a lambda context we want to keep our dynamo client for
        the duration of the lambda execution environment. Using a singleton pattern
        for this case ensures we don't initialize a client for every execution."""
```

Reply:
I knew I had more docs cleanup. Thanks for the re-review.